include/dsn/cpp/auto_codes.h

```diff
@@ -345,6 +346,7 @@ namespace dsn
     DEFINE_ERR_CODE(ERR_NETWORK_INIT_FALED)
     DEFINE_ERR_CODE(ERR_FORWARD_TO_OTHERS)
     DEFINE_ERR_CODE(ERR_OBJECT_NOT_FOUND)
+
```

Comment:
For categorization? If yes, it is best if we can come up some comments for each category.